src/csharp/Microsoft.Spark.Worker/Command/SqlCommandExecutor.cs

```diff
@@ -794,20 +813,21 @@ internal ArrowOrDataFrameGroupedMapCommandExecutor(Version version)
             {
                 FxDataFrame dataFrame = FxDataFrame.FromArrowRecordBatch(input);
                 FxDataFrame resultDataFrame = worker.Func(dataFrame);
+
                 IEnumerable<RecordBatch> recordBatches = resultDataFrame.ToArrowRecordBatches();
 
-                foreach (RecordBatch result in recordBatches)
+                foreach (RecordBatch batch in recordBatches)
                 {
-                    stat.NumEntriesProcessed += result.Length;
+                    RecordBatch final = WrapArrowRecordBatchColumnsInAStruct(batch);
```

Comment:
the name of this var is `final` and its counterpart in `ExecuteArrowGroupedMapCommand` is named `result`.  Should we name them the same for consistency ?

Reply:
Good catch. Fixed it